lib/features/email/presentation/email_view.dart

```diff
@@ -17,7 +17,7 @@ import 'package:tmail_ui_user/features/email/presentation/widgets/sender_and_rec
 import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
 import 'package:filesize/filesize.dart';
 
-class EmailView extends GetWidget {
+class EmailView extends GetView {
```

Comment:
what is the different between `GetView` and `GetWidget`?

Reply:
GetWidget creates a default base controller attached to it. Since EmailController is independent of the view, we should use GetView to avoid unnecessary memory leaks. You can find out more here. `https://pub.dev/packages/get`